cartography/intel/aws/ec2/reserved_instances.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+from botocore.exceptions import ClientError
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_reserved_instances(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
```

Comment:
let's get rid of config parameter in this client creation. Ultimately the goal is to get rid of these config assignments. Instead, users should use environment variables for these settings.

We should do the same in other ec2 resources as well.

Reply:
Done